target/linux/generic/files/include/linux/routerboot.h

```diff
@@ -42,6 +45,9 @@
 #define RB_ID_MAC_ADDRESS_COUNT	14
 #define RB_ID_HW_OPTIONS	21
 #define RB_ID_WLAN_DATA		22
+#define RB_ID_BOARD_IDENTIFIER	23
+#define RB_ID_BOARD_NAME	33
```

Comment:
I'm curious why RB_ID_BOARD_NAME was moved from 5 to 33. Maybe @robimarko can comment?
In order to preserve old behavior, rb_get_board_name() should try the old ID first I think.

Reply:
Yes, I think, we should ask, but there was no 33 parameter in old hard_config revisions, but it seems that there could present both 05 and 33 in new revision, that's why we should try 33 first and return it if it is present. So, I think that my fix is savefull and both backward and forward compatible...

Reply:
fyi,

R1:
```
[ Nobelwind OHVS 1 | node-13 ] strings /dev/mtd1
draH
l;kB
77BA06BD22F5
912UAG-5HPnD
3.41
711Gr100
tDDD
42064206420
ABEG1257pu
<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<| 
LTl|
@FH068LThx
LThx
LThx
LThx
LThx
LThx
LThx
LThx
LThx
<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<|<
|<|<|<|<|<|<|<|<|<|<|<|<|
```


R2:
```
[ node-3 ] strings /dev/mtd1
draH
A2860BFDA4ED
912UAG-5HPnD
6.44.5
711Gr100
tDDD
4206
6662*662,
ABEG1257pu
?K*4
0LTl|
	N^f
6620<
U=.l
6l? 
642.l
@FH068LThx
```

Reply:
![IMG_20200417_142853 (1)](https://user-images.githubusercontent.com/63855545/79569916-d2f79480-80b8-11ea-83a8-07a3f1088b34.jpg)

I have 3 of this boards. Is there any working Image yet (incuding wifi) and if yes can somebody tell me where I can get it from. If possible without patching and compiling